external-crates/move/crates/move-compiler/src/parser/mod.rs

```diff
@@ -143,6 +169,7 @@ fn ensure_targets_deps_dont_intersect(
 }
 
 fn parse_file(
+    vfs: &VfsPath,
```

Comment:
nit, this name is really weird to me
```suggestion
    path: &VfsPath,
```
It's not a virtual file system, just a path